docs: update comment for testing (#5479)
envoyproxy/gateway: site/content/en/latest/boilerplates/testing-the-configuration.md

---
---

{{< tabpane text=true >}}
{{% tab header="With External LoadBalancer Support" %}}

> You can also test the same functionality by sending traffic to the External IP. To get the external IP of the
> Envoy service, run:

Reviewer: ```suggestion
Get the external address of the Envoy service:
```

```shell
export GATEWAY_HOST=$(kubectl get gateway/eg -o jsonpath='{.status.addresses[0].value}')
```

> In certain environments, the load balancer may be exposed using a hostname, instead of an IP address. If so, replace
> `ip` in the above command with `hostname`.
> 

Reviewer: ```suggestion
```

Curl the example app through Envoy proxy:

```shell
curl --verbose --header "Host: www.example.com" http://$GATEWAY_HOST/get
```

{{% /tab %}}
{{% tab header="Without LoadBalancer Support" %}}

Get the name of the Envoy service created the by the example Gateway:

```shell
export ENVOY_SERVICE=$(kubectl get svc -n envoy-gateway-system --selector=gateway.envoyproxy.io/owning-gateway-namespace=default,gateway.envoyproxy.io/owning-gateway-name=eg -o jsonpath='{.items[0].metadata.name}')
```

Port forward to the Envoy service:

```shell
kubectl -n envoy-gateway-system port-forward service/${ENVOY_SERVICE} 8888:80 &
```

Curl the example app through Envoy proxy:

```shell
curl --verbose --header "Host: www.example.com" http://localhost:8888/get
```

> 

Reviewer: We shouldn't need line 38 to line 43.

{{% /tab %}}
{{< /tabpane >}}

> 

Reviewer: Can this be used in other places?

Author: Yep, like quickstart.

Reviewer: Nice, can we also modify other md files the same way?
